eth2prysm/values.yaml

```diff
@@ -0,0 +1,30 @@
+# Default values for eth2prysm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: prysm
+ethereumNetwork: pyrmont
+imageVersion: v1.3.2
+
+beacon:
+  name: beacon
+  label: beacon
+  serviceAccount: beacon
```

Comment:
we don't need to make service account name configurable until there is a use case.

Reply:
Removed. Also removed the usage in service and deployment templates.